src/common/DEFENSIVE_BUFFS.js

```diff
@@ -41,6 +41,10 @@ const DEFENSIVE_BUFFS = [
   {
     spell: SPELLS.SHROUD_OF_PURGATORY,
   },
+  //Frost Death-Knight
+  {
+    spell: SPELLS.PERMAFROST_TALENT,
```

Comment:
I don't think absorbs like this and Soul Leech are worth tracking since they are completely passive.

Reply:
removed both